src/App.vue

```diff
@@ -93,6 +93,14 @@ export default {
       'user',
       'plan'
     ]),
+    ...mapGetters('polling', [
+      'shouldPollTenants',
+      'shouldPollAgents',
+      'shouldPollProjects',
+      'shouldPollFlows',
+      'shouldPollMemberships',
+      'shouldPollMembershipInvitations'
```

Comment:
note that I've created the infrastructure to make all 6 of these global queries use the new polling state but this PR is only for flows & agents